dataclients/kubernetes/ingressv1.go

```diff
@@ -428,10 +430,10 @@ func (ing *ingress) ingressV1Route(
 	}
 
 	var route *eskip.Route
-	if r, ok, err := ing.convertDefaultBackendV1(state, i, ing.forceKubernetesService); ok {
+	if r, ok, err := ing.convertDefaultBackendV1(ic, ing.forceKubernetesService); ok {
 		route = r
 	} else if err != nil {
-		ic.logger.Errorf("error while converting default backend: %v", err)
+		ic.logger.Errorf("Error while converting default backend: %v", err)
```

Comment:
maybe better to stay consistent: `"Failed to convert default backend: %v", err`

Reply:
Fixed